aiplatform/src/main/java/aiplatform/PredictTextEmbeddingsSample.java

```diff
@@ -39,18 +40,24 @@ public static void main(String[] args) throws IOException {
     // https://cloud.google.com/vertex-ai/docs/generative-ai/embeddings/get-text-embeddings
     String endpoint = "us-central1-aiplatform.googleapis.com:443";
     String project = "YOUR_PROJECT_ID";
-    String model = "textembedding-gecko@003";
+    String model = "text-embedding-004";
     predictTextEmbeddings(
         endpoint,
         project,
         model,
         List.of("banana bread?", "banana muffins?"),
-        "RETRIEVAL_DOCUMENT");
+        "QUESTION_ANSWERING",
+        OptionalInt.of(256));
   }
 
   // Gets text embeddings from a pretrained, foundational model.
   public static List<List<Float>> predictTextEmbeddings(
-      String endpoint, String project, String model, List<String> texts, String task)
+      String endpoint,
+      String project,
+      String model,
+      List<String> texts,
+      String task,
```

Comment:
nit: was it possible to keep formatting of these arguments unchanged in order to minimize differences and reduce load on reviewing changes?